docs/tutorial.rst

```diff
@@ -93,8 +93,11 @@ CSVs from URLs are also valid inputs to
 
 ------
 
-It's also possible to add columns from a dictionary, but this option is
-discouraged because dictionaries do not preserve column order.
+It's also possible to add columns from a dictionary. For users< Python 3.6 this method is discouraged because dictionary does not preserve the order.
```

Comment:
Hi, thank you for this! We're in the middle of requiring Python 3.6 for using this library, so can you please make the change to remove this second sentence here?